debian/changelog.dch

```diff
@@ -0,0 +1,9 @@
+on-static-common (1.1.1) unstable; urgency=low
```

Comment:
Ignore this please, this file was unexpected created when I tried dch cmds, I'll delete this.